_posts/2022-02-18-release-notes.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+date: February 18, 2022
+title: "February 18th cloud.gov Change Log"
+excerpt: The cloud.gov team is working on providing change logs so everyone can see new features and updates.
+---
+
+The cloud.gov team is working on providing change logs so everyone can see new features and updates. Highlights this month include changes to several of the platform buildpacks.
+
+# Change Log
+
+## Customer Facing
+
+---
+No changes impacting customers directly this sprint.
```

Comment:
We deployed an new version of stratos from v3 to v4

Reply:
oops..right!